test/controller.go

```diff
@@ -86,7 +76,6 @@ type Informers struct {
 // TestAssets holds references to the controller, logs, clients, and informers.
 type TestAssets struct {
 	Controller *controller.Impl
-	Logs       *observer.ObservedLogs
```

Comment:
This is strange - I remember using Logs to debug a test in the past, before the controller refactor.
I might be nice to have GetLogMessages populate the Logs field here, which would give tests access to logs and would enable testing on seeing a certain log message. 

But I agree since it's not used now we can remove it and add it back if needed in future.